Automattic/jetpack
# Social | Add resharing for classic editor

We're getting requests about resharing in the classic editor. Previously we've de-prioritised any additional work on the classic editor, but WooCommerce is still transitioning away from it, and they are commercial customers. It's probably worth doing if it's low effort.

## Proposed changes:

* Add resharing for classic editor

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Enable classic editor using [classic editor plugin](https://wordpress.org/plugins/classic-editor/)
* Run `jetpack build plugins/social` or `jetpack build plugins/jetpack`
* Create a new post
* Click on "Edit" beside "Jetpack Social"
* Confirm that you don't see the "Share now" button for the posts which are not published yet.
* Publish the post
* Confirm that you now see the "Share now" button
* Use the Share now button
* Confirm that it works and behaves as expected.

<img width="293" alt="Screenshot 2024-06-12 at 2 57 33 PM" src="https://github.com/Automattic/jetpack/assets/18226415/db740801-4a99-4648-bad9-963ff215cfe1">
